src/services/custom/index.js

```diff
@@ -0,0 +1,30 @@
+import * as cache from '../cache';
```

Comment:
This file's functionality hasn't been tested yet and isn't completely finished either.